### PR TITLE
PBL, Convection and Microphysics Update for HR2

### PR DESCRIPTION
**Pull request by Qingfu-Liu**
## Description

This PR is the same as PR#1723, and is created because the ccpp-physics is updated for HR2 (PR#65). The ccpp-physics update in PR#65 includes update for PBL scheme, shallow convection scheme, deep convection scheme and microphysics scheme. The changes of the physics improve hurricane forecast, and CAPE forecast values.

### Input data additions/changes
- [X] No changes are expected to input data.
- [ ] There will be new input data. 
- [ ] Input data will be updated. 

### Anticipated changes to regression tests:
- [ ] No changes are expected to any regression test. 
- [X] Changes are expected to the following tests: 

control, cpld, and hafs cases change result: see the list below in conversation

## Subcomponents involved:
- [ ] AQM
- [ ] CDEPS
- [ ] CICE
- [ ] CMEPS
- [ ] CMakeModules
- [X] FV3
- [ ] GOCART
- [ ] HYCOM
- [ ] MOM6
- [ ] NOAHMP
- [ ] WW3
- [ ] stochastic_physics
- [ ] none

## Commit Queue Checklist:

- [x] Link PR's from all sub-components involved
https://github.com/ufs-community/ccpp-physics/pull/65
https://github.com/NOAA-EMC/fv3atm/pull/653
https://github.com/ufs-community/ufs-weather-model/pull/1731
- [X] Confirm reviews completed in sub-component PR's
- [x] Add all appropriate labels to this PR.
- [x] Run full RT suite on either Hera/Cheyenne with both Intel/GNU compilers
- [ ] Add list of any failed regression tests to "Anticipated changes to regression tests" section.

## Linked PR's and Issues:
https://github.com/ufs-community/ccpp-physics/pull/65
https://github.com/NOAA-EMC/fv3atm/pull/653

## Testing Day Checklist:

- [X] This PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR.
- [ ] Move new/updated input data on RDHPCS Hera and propagate input data changes to all supported systems.

### Testing Log (for CM's):
- RDHPCS
  - Intel
    - [x] Hera
    - [x] Orion
    - [x] Jet
    - [ ] Gaea
    - [x] Cheyenne
  - GNU
    - [x] Hera
    - [x] Cheyenne
- WCOSS2
  - [x] Dogwood/Cactus
  - [x] Acorn
- CI
  - [x] Completed
- opnReqTest
  - [ ] N/A
  - [ ] Log attached to comment

### CODE REVIEW

**Comment by Qingfu-Liu** [comment]
PR#1731 is created and replaces the closed PR#1723. This PR#1731 has merged the ufs-community/ufs-weather-model develop branch into the branch update_HR2.

**Comment by Qingfu-Liu** [comment]
Has a problem using ecflow to run the rt.sh tests, anyone know how to fix this problem?:

+ mkdir /scratch1/NCEPDEV/global/Qingfu.Liu/git/ufs-weather-model/tests/lock
mkdir: cannot create directory '/scratch1/NCEPDEV/global/Qingfu.Liu/git/ufs-weather-model/tests/lock': File exists
+ echo 'Only one instance of rt.sh can be running at a time'
Only one instance of rt.sh can be running at a time

**Comment by jkbk2004** [comment]
`1. Clone your fork and checkout branch that needs syncing:
git clone https://github.com/JoeSmith-NOAA/ufs-weather-model.git ./fork
cd fork
git checkout feature/branch

2. Add upstream info to your clone so it knows where to merge from. The term “upstream” refers to the authoritative rep\
ository from which the fork was created.
git remote add upstream https://github.com/ufs-community/ufs-weather-model.git

3. Fetch upstream information into clone:
git fetch upstream

4. Later on you can update your fork remote information by doing the following command:
git remote update

5. Merge upstream feature/branch (or develop) into your branch:                                                         git merge upstream/feature/branch (or develop)

6. Resolve any conflicts and perform any needed “add”s or “commit”s for conflict resolution.

7. Push the merged copy back up to your fork (origin):
git push origin feature/branch
`

**Comment by jkbk2004** [comment]
@Qingfu-Liu for an example, a general procedure to sync can be
1. Clone your fork and checkout branch that needs syncing:
git clone https://github.com/JoeSmith-NOAA/ufs-weather-model.git ./fork
cd fork
git checkout feature/branch
2. Add upstream info to your clone so it knows where to merge from. The term “upstream” refers to the authoritative rep\
ository from which the fork was created.
git remote add upstream https://github.com/ufs-community/ufs-weather-model.git
3. Fetch upstream information into clone:
git fetch upstream
4. Later on you can update your fork remote information by doing the following command:
git remote update
5. Merge upstream feature/branch (or develop) into your branch:                                                         git merge upstream/feature/branch (or develop)
6. Resolve any conflicts and perform any needed “add”s or “commit”s for conflict resolution.
7. Push the merged copy back up to your fork (origin):
git push origin feature/branch

For submodule component, you can sync up with a similar way.

**Comment by Qingfu-Liu** [comment]
@Jong Kim - NOAA Affiliate ***@***.***> I just sync my forks for
PR#65, PR#653 and PR#1731

**Comment by Qingfu-Liu** [comment]
@Jong Kim - NOAA Affiliate ***@***.***> Thank you very much for the
help. If there is a problem, please let me know

**Comment by jkbk2004** [comment]
[RegressionTests_cheyenne.intel.log](https://github.com/ufs-community/ufs-weather-model/files/11382616/RegressionTests_cheyenne.intel.log)
[RegressionTests_cheyenne.gnu.log](https://github.com/ufs-community/ufs-weather-model/files/11382617/RegressionTests_cheyenne.gnu.log)

**Comment by DeniseWorthen** [comment]
What is going on w/ Cheyenne.gnu?

```
baseline dir = /glade/scratch/epicufsrt/GMTB/ufs-weather-model/RT/NEMSfv3gfs/develop-20230430/GNU/cpld_control_p8
working dir  = /glade/scratch/jongkim/rt-1731-gnu/jongkim/FV3_RT/rt_43398/cpld_control_p8
Checking test 051 cpld_control_p8 results ....
 Comparing sfcf021.tile1.nc ............ALT CHECK......ERROR
```

**Comment by DeniseWorthen** [comment]
I also don't understand how all these coupled tests are giving "alt check" OK. That means generally that the data matches, but the metadata doesn't. If this is actually changing physics, then how is that happening? Are the tests not capturing the changes in the physics?

**Comment by junwang-noaa** [comment]
@DeniseWorthen RT is using "cmp" to compare two files, if the files are different and they are netcdf files, then use "nccmp" (or compare_ncfile.py) to compare all the data fields in the two netcdf files.

**Comment by DeniseWorthen** [comment]
@junwang-noaa I thought the changes in this PR would change the actual forecast results. So I would expect the comparisons to just fail because the data was different.

```
baseline dir = /glade/scratch/epicufsrt/GMTB/ufs-weather-model/RT/NEMSfv3gfs/develop-20230426/INTEL/cpld_control_p8
working dir  = /glade/scratch/jongkim/rt-1731-intel/jongkim/FV3_RT/rt_10944/cpld_control_p8
Checking test 003 cpld_control_p8 results ....
 Comparing sfcf021.tile1.nc ............ALT CHECK......OK
 Comparing sfcf021.tile2.nc ............ALT CHECK......OK
 Comparing sfcf021.tile3.nc ............ALT CHECK......OK
 Comparing sfcf021.tile4.nc ............ALT CHECK......OK
```

For Cheyenne.gnu, I'm not sure what is happening on the sfc021.tile1.nc file. It is giving "error", not "not ok"

**Comment by junwang-noaa** [comment]
I see. @Qingfu-Liu @jkbk2004 Can you check the results? Is this run coming from Qingfu's branch? We need a new baseline.

**Comment by Qingfu-Liu** [comment]
@Jun Wang - NOAA Federal ***@***.***> I am not sure I have an
account on Cheyenne, but I am rerunning the tests on Hera.

**Comment by BrianCurtis-NOAA** [comment]
@DeniseWorthen @junwang-noaa I believe the "ERROR" outputs are because the script and/or command fails, not that any result from the comparison has happened. So we should manually run what's done to see if we can get better information on what part of the command/script is failing.

**Comment by DeniseWorthen** [comment]
@BrianCurtis-NOAA Exactly. The logs always need to be checked for anomalous results.

On cheyenne.intel, I can compare the baseline vs the sfcf021.tile3.nc file from the INTEL case using cprnc and they appear completely different. Why is the comparison appearing as "alt check OK"?

**Comment by DeniseWorthen** [comment]
The issue seems to be the BL date:

```
baseline dir = /glade/scratch/epicufsrt/GMTB/ufs-weather-model/RT/NEMSfv3gfs/develop-20230426/INTEL/cpld_control_p8
working dir  = /glade/scratch/jongkim/rt-1731-intel/jongkim/FV3_RT/rt_10944/cpld_control_p8
```

The current BL date is 0430, right?

**Comment by BrianCurtis-NOAA** [comment]
@Qingfu-Liu 
Update:
- Talk to hera helpdesk to get access to ecflow. Then:
- please run intel RT on Hera with `./rt.sh -e > rt.out 2>&1 &`
- please run gnu RT on Hera with `export RT_COMPILER=gnu && ./rt.sh -e > rt.out 2>&1 &`

**Comment by jkbk2004** [comment]
@Qingfu-Liu if ecflow is issue, let me run hera.gnu

**Comment by Qingfu-Liu** [comment]
@Brian Curtis - NOAA Affiliate ***@***.***>  I am running the
test on Hera, so far everything looks good (still running).

**Comment by Qingfu-Liu** [comment]
@Jong Kim - NOAA Affiliate ***@***.***> I am running  ./rt_1.sh -e >
rt.out 2>&1 &
If you can run the hera.gnu, that will be great. Thanks

**Comment by jkbk2004** [comment]
sounds like we still have exit code issue of nccmp. current exit code setup of d=$? is 1: ERROR, 2 (other than 0): NOT OK, 0: OK. But if change result, it exits with 1. @DusanJovic-NOAA FYI

**Comment by jkbk2004** [comment]
`Exit code 0 is returned for identical files, 1 for different files, and 2 for a fatal error.`

**Comment by Qingfu-Liu** [comment]
Regression tests on Hera completed (./rt_1.sh -e > rt.out 2>&1 &). The log file is here:
/scratch1/NCEPDEV/global/Qingfu.Liu/git/ufs-weather-model/tests/RegressionTests_hera.intel.log

**Comment by BrianCurtis-NOAA** [comment]
@Qingfu-Liu The baseline you are comparing with is old. (in rt.sh look for "BL_DATE"):
```
baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20230418/INTEL/cpld_control_p8_mixedmode
working dir  = /scratch1/NCEPDEV/stmp2/Qingfu.Liu/FV3_RT/rt_13556/cpld_control_p8_mixedmode
 Checking test 001 cpld_control_p8_mixedmode results ....
  Comparing sfcf021.tile1.nc ............ALT CHECK......ERROR
```
 The current branch has the correct BL_DATE (https://github.com/Qingfu-Liu/ufs-weather-model/blob/update_HR2/tests/rt.sh#L471) so I don't know how you are comparing to old baselines. Try (once you get ecflow access on hera):
```
git clone -b update_HR2 https://github.com/Qingfu-Liu/ufs-weather-model --recursive
cd ufs-weather-model/tests
./rt.sh -e > rt.out 2>&1 &
```

**Comment by Qingfu-Liu** [comment]
I just using the command: ./rt.sh -e > rt.out 2>&1 &
in the directory: /scratch1/NCEPDEV/global/Qingfu.Liu/git/ufs-weather-model/tests
I do not know why the script ran incorrectly

**Comment by Qingfu-Liu** [comment]
I just checked out the fork and rerun the regression tests.

**Comment by DusanJovic-NOAA** [comment]
> sounds like we still have exit code issue of nccmp. current exit code setup of d=$? is 1: ERROR, 2 (other than 0): NOT OK, 0: OK. But if change result, it exits with 1. @DusanJovic-NOAA FYI

What kind of error does nccmp return

**Comment by jkbk2004** [comment]
> > sounds like we still have exit code issue of nccmp. current exit code setup of d=$? is 1: ERROR, 2 (other than 0): NOT OK, 0: OK. But if change result, it exits with 1. @DusanJovic-NOAA FYI
> 
> What kind of error does nccmp return

nccmp exit code are 0:  identical, 1: different, 2 (and higher): fatal But the script treating 1 as ERROR (fatal). I applied a fix in rt_util.sh. It looks like working ok now.

**Comment by DusanJovic-NOAA** [comment]
> > > sounds like we still have exit code issue of nccmp. current exit code setup of d=$? is 1: ERROR, 2 (other than 0): NOT OK, 0: OK. But if change result, it exits with 1. @DusanJovic-NOAA FYI
> > 
> > 
> > What kind of error does nccmp return
> 
> nccmp exit code are 0: identical, 1: different, 2 (and higher): fatal But the script treating 1 as ERROR (fatal). I applied a fix in rt_util.sh. It looks like working ok now.

Ok, So there was no actual error from nccmp, just the files were different.

**Comment by jkbk2004** [comment]
> > > > sounds like we still have exit code issue of nccmp. current exit code setup of d=$? is 1: ERROR, 2 (other than 0): NOT OK, 0: OK. But if change result, it exits with 1. @DusanJovic-NOAA FYI
> > > 
> > > 
> > > What kind of error does nccmp return
> > 
> > 
> > nccmp exit code are 0: identical, 1: different, 2 (and higher): fatal But the script treating 1 as ERROR (fatal). I applied a fix in rt_util.sh. It looks like working ok now.
> 
> Ok, So there was no actual error from nccmp, just the files were different.

yes

File: tests/rt_utils.sh [hunk 1]
```diff
@@ -360,13 +360,18 @@ check_results() {
             printf ".......ALT CHECK.."
             if [[ ${MACHINE_ID} =~ orion || ${MACHINE_ID} =~ hera || ${MACHINE_ID} =~ gaea || ${MACHINE_ID} =~ jet || ${MACHINE_ID} =~ cheyenne ]] ; then
               nccmp -d -f -g -B --Attribute=checksum --warn=format ${RTPWD}/${CNTL_DIR}/${i} ${RUNDIR}/${i} > ${i}_nccmp.log 2>&1 && d=$? || d=$?
+              if [$d -ne 0] && [ $d -ne 1 ]; then
```
Comment (DusanJovic-NOAA) [inline tests/rt_utils.sh]: This is not correct syntax. I am getting this error:
```
 Comparing RESTART/20210322.090000.fv_tracer.res.tile1.nc ............ALT CHECK..rt_utils.sh: line 363: [1: command not found        
....NOT OK                                                                                                                           
 Comparing RESTART/20210322.090000.fv_tracer.res.tile2.nc ............ALT CHECK..rt_utils.sh: line 363: [1: command not found        
....NOT OK                                                                                                                           
```
Did you test this in your branch?  Please rewrite it as:
```
if [[ $d -ne 0 && $d -ne 1 ]]; then
```

Comment (jkbk2004) [reply tests/rt_utils.sh]: updated!

**Comment by jkbk2004** [comment]
[RegressionTests_cheyenne.intel.log](https://github.com/ufs-community/ufs-weather-model/files/11401590/RegressionTests_cheyenne.intel.log)
[RegressionTests_cheyenne.gnu.log](https://github.com/ufs-community/ufs-weather-model/files/11401593/RegressionTests_cheyenne.gnu.log)

**Comment by jkbk2004** [comment]
baselines were created ok on cheyenne. intel script further moved to compare for control_c384. I will go ahead to run RTs on cheyenne.

**Comment by DeniseWorthen** [comment]
@jkbk2004 So you were able to create a baseline and compare, but did you ever run against the existing baselines so we know which tests actually fail? That was the issue w/ the previous set of logs, where it turned out the comparison for GNU and INTEL were against different baselines and there were issues about baselines passing when they shouldn't have.

**Comment by jkbk2004** [comment]
` INTEL FAILED TESTS: 
Test cpld_control_p8_mixedmode 001 failed in run_test failed 
Test cpld_control_gfsv17 002 failed in run_test failed 
Test cpld_control_p8 003 failed in run_test failed 
Test cpld_control_qr_p8 005 failed in run_test failed 
Test cpld_2threads_p8 007 failed in run_test failed 
Test cpld_decomp_p8 008 failed in run_test failed 
Test cpld_mpi_p8 009 failed in run_test failed 
Test cpld_control_ciceC_p8 010 failed in run_test failed 
Test cpld_control_c192_p8 011 failed in run_test failed 
Test cpld_control_noaero_p8 013 failed in run_test failed 
Test cpld_debug_p8 015 failed in run_test failed 
Test cpld_debug_noaero_p8 016 failed in run_test failed 
Test control_flake 021 failed in run_test failed 
Test control_latlon 024 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel 025 failed in run_test failed 
Test control_c192 027 failed in run_test failed 
Test control_c384 028 failed in run_test failed 
Test control_c384gdas 029 failed in run_test failed 
Test control_stochy 030 failed in run_test failed 
Test control_lndp 032 failed in run_test failed 
Test control_iovr4 033 failed in run_test failed 
Test control_iovr5 034 failed in run_test failed 
Test control_p8 035 failed in run_test failed 
Test control_qr_p8 037 failed in run_test failed 
Test control_decomp_p8 039 failed in run_test failed 
Test control_2threads_p8 040 failed in run_test failed 
Test control_p8_lndp 041 failed in run_test failed 
Test control_p8_rrtmgp 042 failed in run_test failed 
Test control_p8_mynn 043 failed in run_test failed 
Test merra2_thompson 044 failed in run_test failed 
Test control_csawmg 075 failed in run_test failed 
Test control_csawmgt 076 failed in run_test failed 
Test control_ras 077 failed in run_test failed 
Test control_p8_faster 079 failed in run_test failed 
Test hafs_regional_atm 121 failed in run_test failed 
Test hafs_regional_atm_ocn 123 failed in run_test failed 
Test hafs_regional_atm_wav 124 failed in run_test failed 
Test hafs_regional_atm_ocn_wav 125 failed in run_test failed 
Test hafs_global_multiple_4nests_atm 129 failed in run_test failed 
Test hafs_regional_specified_moving_1nest_atm 130 failed in run_test failed 
Test hafs_regional_storm_following_1nest_atm_ocn 132 failed in run_test failed 
Test hafs_regional_storm_following_1nest_atm_ocn_wav 135 failed in run_test failed 
Test atmwav_control_noaero_p8 157 failed in run_test failed 
Test control_atmwav 158 failed in run_test failed 
Test atmaero_control_p8 159 failed in run_test failed 
Test atmaero_control_p8_rad 160 failed in run_test failed 
Test atmaero_control_p8_rad_micro 161 failed in run_test failed 
 `

**Comment by jkbk2004** [comment]
INTEL FAILED TESTS: 
Test cpld_control_p8_mixedmode 001 failed in run_test failed 
Test cpld_control_gfsv17 002 failed in run_test failed 
Test cpld_control_p8 003 failed in run_test failed 
Test cpld_control_qr_p8 005 failed in run_test failed 
Test cpld_2threads_p8 007 failed in run_test failed 
Test cpld_decomp_p8 008 failed in run_test failed 
Test cpld_mpi_p8 009 failed in run_test failed 
Test cpld_control_ciceC_p8 010 failed in run_test failed 
Test cpld_control_c192_p8 011 failed in run_test failed 
Test cpld_control_noaero_p8 013 failed in run_test failed 
Test cpld_debug_p8 015 failed in run_test failed 
Test cpld_debug_noaero_p8 016 failed in run_test failed 
Test control_flake 021 failed in run_test failed 
Test control_latlon 024 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel 025 failed in run_test failed 
Test control_c192 027 failed in run_test failed 
Test control_c384 028 failed in run_test failed 
Test control_c384gdas 029 failed in run_test failed 
Test control_stochy 030 failed in run_test failed 
Test control_lndp 032 failed in run_test failed 
Test control_iovr4 033 failed in run_test failed 
Test control_iovr5 034 failed in run_test failed 
Test control_p8 035 failed in run_test failed 
Test control_qr_p8 037 failed in run_test failed 
Test control_decomp_p8 039 failed in run_test failed 
Test control_2threads_p8 040 failed in run_test failed 
Test control_p8_lndp 041 failed in run_test failed 
Test control_p8_rrtmgp 042 failed in run_test failed 
Test control_p8_mynn 043 failed in run_test failed 
Test merra2_thompson 044 failed in run_test failed 
Test control_csawmg 075 failed in run_test failed 
Test control_csawmgt 076 failed in run_test failed 
Test control_ras 077 failed in run_test failed 
Test control_p8_faster 079 failed in run_test failed 
Test hafs_regional_atm 121 failed in run_test failed 
Test hafs_regional_atm_ocn 123 failed in run_test failed 
Test hafs_regional_atm_wav 124 failed in run_test failed 
Test hafs_regional_atm_ocn_wav 125 failed in run_test failed 
Test hafs_global_multiple_4nests_atm 129 failed in run_test failed 
Test hafs_regional_specified_moving_1nest_atm 130 failed in run_test failed 
Test hafs_regional_storm_following_1nest_atm_ocn 132 failed in run_test failed 
Test hafs_regional_storm_following_1nest_atm_ocn_wav 135 failed in run_test failed 
Test atmwav_control_noaero_p8 157 failed in run_test failed 
Test control_atmwav 158 failed in run_test failed 
Test atmaero_control_p8 159 failed in run_test failed 
Test atmaero_control_p8_rad 160 failed in run_test failed 
Test atmaero_control_p8_rad_micro 161 failed in run_test failed 

GNU FAILED TESTS: 
Test control_c48 001 failed in run_test failed 
Test control_stochy 002 failed in run_test failed 
Test control_ras 003 failed in run_test failed 
Test control_p8 004 failed in run_test failed 
Test control_flake 005 failed in run_test failed 
Test control_diag_debug 023 failed in run_test failed 
Test control_ras_debug 031 failed in run_test failed 
Test control_stochy_debug 032 failed in run_test failed 
Test control_debug_p8 033 failed in run_test failed 
Test control_wam_debug 039 failed in run_test failed 
Test cpld_control_p8 051 failed in run_test failed 
Test cpld_control_nowave_noaero_p8 052 failed in run_test failed 
Test cpld_debug_p8 053 failed in run_test failed

**Comment by jkbk2004** [comment]
@Qingfu-Liu can you confirm these failed cases that this pr is intended for ?

**Comment by BrianCurtis-NOAA** [comment]
We need a confirmation/discussion with @Qingfu-Liu on the failed tests before we start AutoRT.

**Comment by Qingfu-Liu** [comment]
@Brian Curtis - NOAA Affiliate ***@***.***> @Jong Kim - NOAA
Affiliate ***@***.***> This PR#1731 changes physics (PBL,
deep/shallow convection and microphysics). The regression test  will fail
(or results will change) for any suite using physics

**Comment by Qingfu-Liu** [comment]
@Brian Curtis - NOAA Affiliate ***@***.***> @Jong Kim - NOAA
Affiliate ***@***.***> we need create new baseline for this PR#1731

**Comment by Qingfu-Liu** [comment]
@Brian Curtis - NOAA Affiliate ***@***.***> @Jong Kim - NOAA
Affiliate ***@***.***> I should clarify that results will change for
any suites using GFS physics. Some of suites may not use GFS physics, so
for those suites, result will not change

**Comment by FernandoAndrade-NOAA** [comment]
Please see the attached logs for jenkins ci, ORTs passed.
[ufs-weather-model » ort-docker-pipeline » PR-1731 #1 Console [Jenkins].pdf](https://github.com/ufs-community/ufs-weather-model/files/11422367/ufs-weather-model.ort-docker-pipeline.PR-1731.1.Console.Jenkins.pdf)

**Comment by junwang-noaa** [comment]
@Qingfu-Liu would you please add the code changes in issue #1662 in your CCPP PR? these are print line changes and have no impact on results. Thanks

**Comment by Qingfu-Liu** [comment]
@Jun Wang - NOAA Federal ***@***.***> @Jong Kim - NOAA Affiliate
***@***.***> I just fixed  issue #1662
<https://github.com/ufs-community/ufs-weather-model/issues/1662> inside the
PR#65

**Comment by BrianCurtis-NOAA** [comment]
regional_atmaq_faster failed on both Acorn and WCOSS2, I have disabled them, and have filed issue #1742.

**Comment by BrianCurtis-NOAA** [comment]
Running comparison RT's now on Acorn/Cactus.

**Comment by BrianCurtis-NOAA** [comment]
Acorn: I left the rsync overnight to run comparisons this AM and ran into space issues that are now resolved as of this AM. I resumed the rsync of baselines and i'll start comparisons shortly after that completes.

**Comment by jkbk2004** [comment]
All tests are done. We can start merging process.

**Comment by jkbk2004** [comment]
@Qingfu-Liu fv3 pr was merged. correct hash is https://github.com/NOAA-EMC/fv3atm/commit/160b422dc1d294623cc8ebfabfbe7d28d7c65c62. Can you update the hash and revert change in gitmodules?

**Review by jkbk2004 — APPROVED** [review]

**Review by BrianCurtis-NOAA — APPROVED** [review]

**Comment by Qingfu-Liu** [comment]
@Jong Kim - NOAA Affiliate ***@***.***> I just pushed the changes
back to the fork

**Review by DusanJovic-NOAA — APPROVED** [review]

**Pull request merged**